exonum-light-client/src/main/java/com/exonum/client/request/BlockFilteringOption.java

```diff
@@ -21,8 +21,15 @@
  */
 public enum BlockFilteringOption {
   /**
-   * Skip empty blocks (containing no transactions).
-   * Only non-empty blocks will be returned in a response.
+   * Skip empty blocks (containing no transactions). Only non-empty blocks will be returned
+   * in a response.
+   *
+   * <p>Requesting the <em>server</em> to skip empty blocks when the client does not need them
+   * allows to save bandwidth and complete the request faster if a large number of blocks
+   * is requested. However, some blockchains never have empty blocks, in which case this option
+   * has no effect (e.g., an active <a href="https://exonum.com/doc/version/latest/advanced/time/">
```

Comment:
Specify a version here?

Reply:
I left it out because the light client is not based on a particular version of Exonum, and may be compatible with several versions (on the other hand, not necessarily the latest :thinking: ). Ideally, it must be the latest version that the _light client_ supports, but I doubt the trouble to keep it in sync is worth it. In EJB it is simpler because it is based on a particular version.